go.mod

```diff
@@ -4,34 +4,65 @@ go 1.18
 
 require (
 	github.com/ethereum/go-ethereum v1.10.17
+	github.com/gofrs/uuid v3.3.0+incompatible
```

Comment:
Why not more common github.com/google/uuid? At least it is go.mod-friendly.

Reply:
fixed